package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "backend-swe-interview-questions",
+  "version": "0.0.1",
+  "description": "Backend SWE interview questions",
+  "homepage": "https://github.com/tamhoang1412/backend-swe-interview-questions#readme",
+  "bugs": {
+    "url": "https://github.com/tamhoang1412/backend-swe-interview-questions/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tamhoang1412/backend-swe-interview-questions.git"
+  },
+  "license": "MIT",
+  "author": "Tam Hoang",
+  "scripts": {
+    "prettier": "prettier '{,!(node_modules)/**/}*.md' --check",
+    "prettier:fix": "npm run prettier -- --write",
+    "sort": "npx sort-package-json"
```

Comment:
Do we need this sort script?

Reply:
It's just a trick I recently have. In the future, we may install dependencies by editing `package.json` directly (e.g: integrate `lint-stage`), or update some fields in that file. To make `package.json` organized, just need to run `npm run sort`, it will sort all fields recursively.
Let's think of it as prettier, but for `package.json`

Reply:
Moreover, it use `npx`, so it does not cost any disk storage to the users (except some bytes for the scripts 😂😂)